### PR TITLE
Add strictSSL field for npm

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Should be merged after https://github.com/saucelabs/saucectl/pull/289

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by sweeneyskirt-sl — APPROVED** [review]

**Comment by sweeneyskirt-sl** [comment]
Hi @tianfeng92  -- This PR has `draft` status. Are you leaving it open for now to apply more changes, or is it ready to merge?

File: docs/testrunner-toolkit/configuration/common-syntax.md [hunk 1]
```diff
@@ -197,6 +199,17 @@ __Example__:
     "@cypress/react": "^5.0.1"
 ```
 
+### `strictSSL`
+
+__Description__: Whether or not to do SSL key validation when making requests to the registry via https. If this value is not set, npm will set it to `true` by default.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/common-syntax.md:204]:
```suggestion
__Description__: Instructs npm to perform SSL key validation when making requests to the registry via https (`true`) or not (`false`). Defaults to `true` when not set.
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]
One suggestion to make it clear which value corresponds to which behavior. Thanks!

File: docs/testrunner-toolkit/configuration/common-syntax.md (hunk 1 above)
Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/common-syntax.md:204]: Thanks!

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**